README.rst

```diff
@@ -82,8 +82,10 @@ the PyFluent-Parametric documentation.
 
 .. code:: python
 
+   import ansys.fluent.core as pyfluent
    from ansys.fluent.parametric import ParametricStudy
-   study_1 = ParametricStudy(session.solver.root.parametric_studies).initialize()
+   solver_session = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
```

Comment:
We can simplify this a bit more: precision is by double by default, processot_count is not relevant. Also rename study_1 -> study.

Reply:
Done. Thank you.